app/assets/stylesheets/govuk_publishing_components/components/_step-by-step-nav.scss

```diff
@@ -77,22 +82,38 @@ $top-border: solid 2px govuk-colour("mid-grey", $legacy: "grey-3");
 }
 
 .gem-c-step-nav__button--title {
-  @include step-nav-font(19, $weight: bold, $line-height: 1.4);
+  @include step-nav-font(19, $weight: bold, $line-height: 1.2);
   display: inline-block;
-  padding: 0;
+  padding: govuk-spacing(1) 0 0;
   text-align: left;
   color: govuk-colour("black");
+  width: 100%;
 
   .gem-c-step-nav--large & {
     @include step-nav-font(19, $tablet-size: 24, $weight: bold, $line-height: 1.4);
+
+    @include govuk-media_query($from: tablet) {
+      padding-top: govuk-spacing(2);
+    }
+  }
+
+  &:focus {
+    box-shadow: 0 -4px govuk-colour("black");
+
+    .gem-c-step-nav__toggle-link {
+      color: govuk-colour("black");
+    }
+
+    @include step-nav-chevron-focus-state;
   }
 }
 
 .gem-c-step-nav__button--controls {
   @include step-nav-font(14, $line-height: 1);
   position: relative;
   z-index: 1; // this and relative position stops focus outline underlap with border of accordion
-  padding: .5em 0 1.5em;
+  margin: .5em 0 1.5em;
+  padding: 0 0 govuk-spacing(1);
 
   &:hover {
```

Comment:
🤔  [Did this change](https://govuk-publis-step-by-st-wvvclh.herokuapp.com/component-guide/step_by_step_nav)? There is an underline on hover for the accordion for the "Show" copy otherwise there's no visual cue for this state?

Reply:
Don't know where that's gone, I'll investigate and put it back in.